File: 10818.c
#include<stdio.h>
int arr[1000000];
int main() {
	int MAX;
	int MIN;
	int i = 0;
	int N;
	scanf("%d", &N);

	for (i = 0; i < N; i++) {
		scanf("%d", &arr[i]);
	}

	MAX = arr[0];
	MIN = arr[0];
	for (int i = 0; i < N; i++) {
		if (MAX <= arr[i]) {
			MAX = arr[i];
		}
		if (MIN >= arr[i]) {
			MIN = arr[i];
		}
	}

	printf("%d %d", MIN, MAX);

}

File: java/10818.java
import java.util.Scanner;
class Main{
    public static void main(String args[]){
        int N,tmp,Max,Min;
        Scanner scan = new Scanner(System.in);
        N = scan.nextInt();
        scan.nextLine();
        
        String list[] = scan.nextLine().split(" ");
        
        Max = Min = Integer.parseInt(list[0]);
        for(int i =0; i < N; i++){
            tmp = Integer.parseInt(list[i]);
            if(tmp > Max)
                Max = tmp;
            else if(tmp < Min)
                Min = tmp;
        }
        System.out.println(Min+" "+ Max);
    }
}
